src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/mobile/create_interaction_choice_set_test.cc

```diff
@@ -837,7 +837,7 @@ TEST_F(CreateInteractionChoiceSetRequestTest,
   smart_objects::SmartObject choice_set_id(smart_objects::SmartType_Null);
 
   EXPECT_CALL(app_mngr_, application(kConnectionKey))
```

Comment:
@ychernysheva its a good candidate for `ON_CALL`

Reply:
Done in a90a2bb